examples/check_examples

```diff
@@ -159,11 +205,21 @@ cd -
 
 echo "Running gocean/eg2"
 cd gocean/eg2
-# We don't do the full compile because that requires a Fortran compiler
-# and the dl_esm_inf library
-${PSYCLONE} -api "gocean1.0" -s ./acc_transform.py alg.f90
-${PSYCLONE} -api "gocean1.0" -s ./acc_prof_transform.py alg.f90
-cd -
+if [ $COMPILE -eq 1 ]
+then
+    # Only do the full compile if requested (because it requires a Fortran
+    # compiler). Defaults to using gfortran.
```

Comment:
Is the "defaults to PSyclone" a relevant point here?

Reply:
I think so (presumably you meant "Defaults to using gfortran"?). I've extended the comment to say that F90 overrides this.

Reply:
What I think this needs is information on where the default is set and how to change it.

Reply:
I've spelt it out a bit more.